kubernetes/kustomization.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - namespace.yaml # Create a namespace (particular-platform-example)
+  - https://github.com/rabbitmq/cluster-operator/releases/latest/download/cluster-operator.yml # Using the RabbitMQ cluster operator
```

Given the current version of the transport relies on 4.x of RabbitMQ and the current latest version has 4.1 with management plugin, I would rather hardcode the exact version for now instead of rolling forward to latest otherwise that could break at any time.

```suggestion
  - https://github.com/rabbitmq/cluster-operator/releases/download/v2.14.0/cluster-operator.yml # Using the RabbitMQ cluster operator
```

---

That being said I see we are already using that approach in the readme so we don't necessarily have to change it here either.

---

Yeah, I just copied that from the readme.